draft-quic-atsss-reqs.mkd

```diff
@@ -619,7 +631,28 @@ Both proposals rely upon single QUIC connections and inherit thus the same issue
 
 In 3GPP architectures, a variety of encapsulations are already used to convey user data. The use of QUIC as a tunneling method for ATSSS purposes will thus add an extra overhead. This overhead is even exacerbated when the user data is forwarded over a non-3GPP network access given than an IPsec tunnel is used to graft the UE to the 5G Core Network. For example, the overhead without taking into account the QUIC tunnel is at least 142 octets for an IPv6 packet forwarded over a non-3GPP network access. As a reminder, ATSSSv1 does not add any overhead. 
 
-More investigation is required to assess whether the ATSSSv2 overhead is an issue. Solutions to optimize that overhead might be considered, if needed. 
+More investigation is required to assess whether the ATSSSv2 overhead is an issue. Solutions to optimize that overhead might be considered, if needed.
+
+## Reordering support for (MP-)QUIC splitting mode
+Both tunnel solutions in {{sec-quic-tunnel}} and {{sec-mpquic-tunnel}} allow the
+splitting mode for simultaneously send data over multiple paths. The respective
+receiver side of the tunnel termination points has to deal with the unreliable
+transport character of the Unreliable Datagram Extension for QUIC and a possible
+re-assembly of the data stream. In practice, it is assumed that some sort of
+re-assembly has to be applied. Different to MPTCP, which is reliable by
+nature and has just to wait for missing segments, such re-assembly mechanisms
+have to account for the fact that packet loss may occur for any of the pathes
+within the QUIC tunnel. Another issue is the packet reordering introduced when a
+QUIC tunnel communication is split across paths with disjoint latencies. In
+theory, most applications or transport protocols certainly have to deal with
```

Comment:
Hi Med,

even if I adopt your suggestion, I think we should point to transport protocol as wells. With QUIC tunneling we possible intend to encapsulate transport protocols like QUIC itself, which have demands on ordering.

Reply:
Hi Markus,

I'm considering QUIC tunneling as an application over UDP, hence my suggested modification.

Feel free to maintain your original text if you think it is better.

Thanks.

Reply:
Hi Med,

can you check again https://github.com/obonaventure/draft-quic-atsss-reqs/pull/88/commits/ff3e13fa5efb9d4892edf7eab01b939abdd7fd2f. Tried to make it clearer with the term "piggybacked"

Reply:
Looks better, thanks. 

If you can maintain "some applications are sensitive to reordering", this would even be great. I'm thinking about VoIP and the like.

Reply:
Added some examples of services potentially sensitive to re-ordering: https://github.com/obonaventure/draft-quic-atsss-reqs/pull/88/commits/a20010d67cf7b227709ad537942256dddb274d53